dc-cudami-editor/src/components/PagedIdentifiableList.jsx

```diff
@@ -232,106 +245,118 @@ class PagedIdentifiableList extends Component {
+                disabledClassName="disabled"
+                forcePage={this.state.pageNumber}
+                marginPagesDisplayed={1}
+                nextClassName="page-item"
+                nextLabel="&raquo;"
+                nextLinkClassName="page-link"
+                onPageChange={this.updatePage}
+                pageClassName="page-item"
+                pageCount={this.state.numberOfPages}
+                pageLinkClassName="page-link"
+                pageRangeDisplayed={5}
+                previousClassName="page-item"
+                previousLabel="&laquo;"
+                previousLinkClassName="page-link"
+              />
+            )}
+            <span className="ml-2">
+              {t(`totalElements.${type}s`, {count: this.state.totalElements})}
+            </span>
+            <Table bordered className="mb-0" hover responsive size="sm" striped>
+              <thead>
+                <tr>
+                  <th className="text-right">
+                    <FaHashtag />
+                  </th>
+                  <th className="text-center">
+                    <FaImage />
+                  </th>
+                  <th>{t('label')}</th>
+                  <th>{t('identifiers')}</th>
+                  <th className="text-center">{t('lastModified')}</th>
+                  <th className="text-center">{t('actions')}</th>
+                </tr>
+              </thead>
+              <tbody>
+                {identifiables.map((identifiable, index) => (
+                  <IdentifiableListItem
+                    apiContextPath={apiContextPath}
+                    enableMove={enableMove}
+                    enableRemove={enableRemove}
+                    identifiers={identifiable.identifiers}
+                    identifierTypes={identifierTypes}
+                    index={index + 1 + pageNumber * this.pageSize}
+                    key={identifiable.uuid}
+                    label={identifiable.label[activeLanguage]}
+                    lastModified={identifiable.lastModified}
+                    onMove={() => {
+                      this.toggleModal('moveAttachedIdentifiable')
+                      this.setState({moveIndex: index})
+                    }}
+                    onRemove={() => {
+                      this.toggleModal('removeAttachedIdentifiable')
+                      this.setState({removeIndex: index})
+                    }}
+                    parentType={parentType}
+                    previewImage={identifiable.previewImage}
+                    previewImageRenderingHints={
+                      identifiable.previewImageRenderingHints
+                    }
+                    showEdit={showEdit}
+                    type={type}
+                    uiLocale={uiLocale}
+                    uuid={identifiable.uuid}
+                  />
+                ))}
+              </tbody>
+            </Table>
+            {identifiables.length > 0 && (
+              <ReactPaginate
```

Comment:
Wenn das identisch oben und unten ist, unbedingt in eine Variable auslagern!

```javascript
// In render() definieren
const pagination = <ReactPaginate ... />

// Im  JSX re-usen:
{identifiables.length > 0 && pagination}
```
```

Reply:
Ist umgebaut, schaut deutlich kompakter aus :+1: